packages/dds/task-manager/src/taskManager.ts

```diff
@@ -312,6 +312,34 @@ export class TaskManager extends SharedObject<ITaskManagerEvents> implements ITa
         return lockAcquireP;
     }
 
+    public subscribeToTask(taskId: string) {
```

Comment:
Probably also eventually want to hit all of these with inheritDocs at some point (similar to our other DDSs), but fair to defer that to another PR.